pubspec.yaml

```diff
@@ -0,0 +1,20 @@
+name: flame_scrolling_sprite
+description: Flame package adding a simple scrolling sprite to make it easy to repeat the same image over time.
+version: 0.0.1
+author:
```

Comment:
I think this tag was deprecated

Reply:
yeah, yet the template stills creates with this, I am going to remove it.